website/content/api-docs/jobs.mdx

```diff
@@ -547,6 +557,62 @@ $ curl \
   - `batch`: Allocations are intended to exit.
   - `system`: Each client gets an allocation.
 
+## Read Job Submission
+
+This endpoint reads original source information about a specific version of a single
+job. The data this endpoint provides is only available if it was provided with the
+original job during job registration.
```

Comment:
Should we include details here about the maximum number of submissions to expect to remain?